### DISCUSSION TITLE
please Solve problem Undefined array key "id"

### DISCUSSION THREAD

**Discussion by saeedalaloul** [livewire/livewire#3799] [7. Bugs (archived)] [answered]
![Capture](https://user-images.githubusercontent.com/42633968/133888110-2c899085-84db-47ea-9b8d-65ba3e59b655.PNG)

**Comment by joshhanley** [comment, marked as answer]
@saeedalaloul please see this pinned discussion for how to fix this #3769

Hope this helps!

**Comment by saeedalaloul** [comment]
Thank you, I fixed the error

**Reply by joshhanley** [reply]
No worries 🙂
